x/distribution/keeper/delegation_test.go

```diff
@@ -216,6 +216,7 @@ func TestCalculateRewardsMultiDelegator(t *testing.T) {
 	// create validator with 50% commission
 	tstaking.Commission = stakingtypes.NewCommissionRates(sdk.NewDecWithPrec(5, 1), sdk.NewDecWithPrec(5, 1), sdk.NewDec(0))
 	tstaking.CreateValidator(valAddrs[0], valConsPk1, 100, true)
+	app.ExecuteEpoch(ctx)
```

Comment:
Thoughts on doing this before every endblock, versus making `tstaking` be initialized to have an epoch time of `1` block?

Reply:
This is for fixing existing tests, we will add more tests which has several endblockers in an epoch.